src/modules/commander/commander_params.c

```diff
@@ -243,14 +243,16 @@ PARAM_DEFINE_FLOAT(COM_DISARM_LAND, 2.0f);
 PARAM_DEFINE_FLOAT(COM_DISARM_PRFLT, 10.0f);
 
 /**
- * GPS preflight check
+ * Arming without GNSS configuration
  *
- * Measures taken when a check defined by EKF2_GPS_CHECK is failing.
+ * Configures whether arming is allowed without GNSS, for modes that require a global position
+ * (specifically, in those modes when a check defined by EKF2_GPS_CHECK fails).
+ * The settings deny arming and warn, allow arming and warn, or silently allow arming.
  *
  * @group Commander
  * @value 0 Deny arming
- * @value 1 Warning only
- * @value 2 Disabled
+ * @value 1 Allow arming (with warning)
+ * @value 2 Allow arming (no warning)
```

Comment:
The inline comments where this is used hint that you allow arming if you have another source of position, such as VIO. When would you choose to warn or not to warn if you have such a system?